src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -345,7 +345,7 @@
     <Compile Include="$(CommonPath)Interop\FreeBSD\Interop.Process.GetProcInfo.cs"
              Link="Common\Interop\FreeBSD\Interop.Process.GetProcInfo.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsUnknownUnix)' == 'true'">
+  <ItemGroup Condition="'$(TargetsUnknownUnix)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">
```

Comment:
If we actually need to treat iOS and tvOS like unknown unix (i.e. the whole System.Diagnostics.Process should be disregarded), we can simply delete their specific frameworks: https://github.com/dotnet/runtime/blob/5fda426afd6cb2ca3c226d4f471bb8bed37cbfbc/src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj#L5
and anything matching ios/tvos in this file. That would be more cleaner. :)

Reply:
@rolfbjarne Do you happen to know if there is such intention?

Reply:
Things like `System.Diagnostics.Process.GetCurrentProcess ().Id` still works fine, so no, we can't remove the entire Process class.